### PR TITLE
feat(cli): Added new info command

### LINKED ISSUE #1838: [CLI]: Having a dedicated command to show up local environment information
Instead of asking the user to manually provide information concerning his local environment, we could employ a dedicated command `codeceptjs info` which shows up the respective information and the `ISSUE_TEMPLATE` can be updated accordingly.

### PR DESCRIPTION
**Pull request by jamesgeorge007**
closes #1838

## Proposed API

`codeceptjs info`

```bash
Environment information:-


  System:
    OS: Linux 5.0 Ubuntu 19.04 (Disco Dingo)
    CPU: (4) x64 Intel(R) Core(TM) i5-7200U CPU @ 2.50GHz
  Binaries:
    Node: 8.11.4 - /usr/bin/node
    Yarn: 1.17.0-0 - /usr/local/bin/yarn
    npm: 6.10.3 - /usr/local/bin/npm
  Browsers:
    Chrome: 76.0.3809.87
    Firefox: 68.0.1
  npmGlobalPackages:
    codeceptjs: 2.2.1
```

### CODE REVIEW

**Comment by jamesgeorge007** [comment]
cc @DavertMik

**Comment by DavertMik** [comment]
Nice! Could we have it as command "codeceptjs report" ?
So it would be used to create issues or ask questions. This command won't just gather PR it will also recommend our Slack channel, Discussion board, or GitHub issues.

And becsides environment variables it should also collect configuration (with currently enabled helpers) and format them in markdown format. 

So here how I see it:

I run 

```
codeceptjs report
```

Then an information is collected and printed on a screen. A message says:

* If you have a question ask it in our Slack https://join.slack.com/t/codeceptjs/shared_invite/enQtMzA5OTM4NDM2MzA4LTNiMTE5YzNlMjQyNWYwZjAzODE3ZTI1MmVkYjY0YTYyNDZmMWJjM2VlODg3NTQzYzM2OGE5ZmIwODFjODk1MzI (url should be minified)
* Or ask on our discussion board https://codecept.discourse.group/
* If you found a bug report it to GitHub issues https://github.com/Codeception/CodeceptJS/issues

Copy information printed above to your message so we could help you.

---

SMth like that :) Is that possible, @jamesgeorge007 ?

**Comment by jamesgeorge007** [comment]
IMO showing up all those information, would bloat the CLI, essentially having a negative impact on the overall UX. I would prefer having all those information listed within the `ISSUE/PR` templates instead.
@DavertMik thoughts :thinking:

**Comment by DavertMik** [comment]
@jamesgeorge007 Sorry for a misunderstanding. I don't mean to collect lot of information. All we need is 

```

  System:
    OS: Linux 5.0 Ubuntu 19.04 (Disco Dingo)
    CPU: (4) x64 Intel(R) Core(TM) i5-7200U CPU @ 2.50GHz
  Binaries:
    Node: 8.11.4 - /usr/bin/node
    Yarn: 1.17.0-0 - /usr/local/bin/yarn
    npm: 6.10.3 - /usr/local/bin/npm
  Browsers:
    Chrome: 76.0.3809.87
    Firefox: 68.0.1
  npmGlobalPackages:
    codeceptjs: 2.2.1
```
+ enabled helpers and plugins. They should be taken from `codecept.conf.js`. 

Also there should be links to Slack and Forum. Because lots of problems are not bugs so they could be solved faster in chat.

**Comment by PeterNgTr** [comment]
@jamesgeorge007 could you conclude this pr? :) I think it's quite useful for anyone when they submit their issue.

File: bin/codecept.js [hunk 1]
```diff
@@ -155,6 +156,20 @@ program.command('run-multiple [suites...]')
...
+  .action(() => {
+    console.log('\n Environment information:-\n');
+    envinfo
+      .run({
+        System: ['OS', 'CPU'],
+        Binaries: ['Node', 'Yarn', 'npm'],
+        Browsers: ['Chrome', 'Edge', 'Firefox', 'Safari'],
+        npmGlobalPackages: ['codeceptjs'],
+      })
+      .then(console.log);
```
Comment (PeterNgTr) [inline bin/codecept.js]: do we really need this?

Comment (jamesgeorge007) [reply bin/codecept.js]: Are you asking about browser information?

Comment (PeterNgTr) [reply bin/codecept.js]: no, the `.then(console.log);` line

Comment (jamesgeorge007) [reply bin/codecept.js]: Logs the respective data (env-info) as [here](https://github.com/Codeception/CodeceptJS/pull/1839#issue-306217645)
It returns a `Promise`

**Comment by PeterNgTr** [comment]
Hey @jamesgeorge007 could you update docs to reflect what this new command does. It would be helpful to everyone.

**Comment by PeterNgTr** [comment]
@jamesgeorge007 could you add documentation how this command can be used?

File: docs/commands.md [hunk 2]
```diff
@@ -197,3 +197,11 @@ Prints all available methods of `I` to console
 ```sh
 npx codeceptjs list
 ```
+
+## Local Environment Information
+
+Prints debugging information concerning the local environment
+
+```sh
+codeceptjs info
```
Comment (PeterNgTr) [inline docs/commands.md]: probably this?
```sh
npx codeceptjs info
```

File: bin/codecept.js [hunk 3]
```diff
@@ -155,6 +156,20 @@ program.command('run-multiple [suites...]')
...
+program.command('info')
+  .description('Print debugging information concerning the local environment')
+  .action(() => {
+    console.log('\n Environment information:-\n');
+    envinfo
+      .run({
+        System: ['OS', 'CPU'],
+        Binaries: ['Node', 'Yarn', 'npm'],
+        Browsers: ['Chrome', 'Edge', 'Firefox', 'Safari'],
+        npmGlobalPackages: ['codeceptjs'],
```
Comment (PeterNgTr) [inline bin/codecept.js]: as @DavertMik aforementioned, may you add the `helpers` and `plugins` as well. Those info are also useful.

Comment (jamesgeorge007) [reply bin/codecept.js]: Can you gimme a better picture? we have got the provision to use glob patterns to refer em.

Comment (PeterNgTr) [reply bin/codecept.js]: maybe showing smt like this:

```sh
   System:
    OS: Linux 5.0 Ubuntu 19.04 (Disco Dingo)
    CPU: (4) x64 Intel(R) Core(TM) i5-7200U CPU @ 2.50GHz
  Binaries:
    Node: 8.11.4 - /usr/bin/node
    Yarn: 1.17.0-0 - /usr/local/bin/yarn
    npm: 6.10.3 - /usr/local/bin/npm
  Browsers:
    Chrome: 76.0.3809.87
    Firefox: 68.0.1
  npmGlobalPackages:
    codeceptjs: 2.2.1
 codeceptjsConfigs: {
  helpers: {
    Puppeteer: {
      url: 'https://www.pizza.de',
      windowSize: "1440x700",
      show: true
    },
    plugins: {
      allure: {
        outputDir: "report",
        enabled: true
     }
  }
```

@DavertMik may you expose more?

**Comment by DavertMik** [comment]
Merging for a request of @PeterNgTr

**Pull request merged**